src/cargo/core/features.rs

```diff
@@ -210,11 +209,15 @@ impl Edition {
         Self::Edition2018,
         Self::Edition2021,
         Self::Edition2024,
+        Self::EditionFuture,
     ];
     /// Possible values allowed for the `--edition` CLI flag.
     ///
     /// This requires a static value due to the way clap works, otherwise I
     /// would have built this dynamically.
+    ///
+    /// This does not include `future` since we don't need to create new
+    /// projects with it.
```

Comment:
nit: we use packages 📦 right?

```suggestion
    /// packages with it.
```